### DISCUSSION TITLE
Improve docs: internationalization, search, etc

### DISCUSSION THREAD

**Discussion by Akimotorakiyu** [josdejong/mathjs#2765] [Ideas]
Translate into other countries' languages to expand the scope of the community and gain more support.

Use [vuepress](https://vuepress.vuejs.org/) to create multilingual documents like [the doc of vuepres's](https://vuepress.vuejs.org/).

Or other ways.

@josdejong  Please internationalization!

**Comment by josdejong** [comment]
Thanks for your suggestion @SilverLeaves , makes sense. 

So far, the docs are automatically generated from comments in the source code. However, we're working on making the code base modular (see #71) which means code for a single function will not be in a single file and function anymore. So it makes sense to rethink this and see if we can move the docs into a separate site. 

Let's keep it open for now whether to use vuepress or an other tool, I guess there are various options with each their pros and cons.

Anyone interested in looking into this?

**Comment by Akimotorakiyu** [comment]
Can we consider integrating multilingual document generation tools into this project? I'm sure it does not conflict with the current system, it can build multilingual documents with existing documents.

**Comment by josdejong** [comment]
Yes indeed, good point to look into extending the current solution!

**Comment by josdejong** [comment]
I'm changing the title of the topic so it covers improving the docs in general, and multilingual is one feature that we have to take into account.

**Comment by Akimotorakiyu** [comment]
Looking forward to multilingual solutions
